feat: add instructions to set directory permissions on manually installs (#4413)
influxdata/docs-v2: content/influxdb/v2.4/install.md

    If you do not move the `influxd` binary into your `$PATH`, prefix the executable
    `./` to run it in place.

{{< expand-wrapper >}}
{{% expand "<span class='req'>Recommended</span> – Set appropriate directory permissions" %}}

> For added security, it is recommended to set the permissions on the influxdb data-dir to not be world readable to prevent unwanted access to data.

Reviewer: ```suggestion
To prevent unwanted access to data, we recommend setting the permissions on the influxdb `data-dir` to not be world readable.
```

Example:

```shell
> chmod 0750 ~/.influxdbv2
```

{{% /expand %}}

> {{< /expand-wrapper >}}

Reviewer 2: @jeffreyssmith2nd You can put two `expand` blocks in one `expand-wrapper` block. Instead of having two separate `expand-wrappers` here, would you move the `expand` block into the one below, or vice versa:

```html
{{< expand-wrapper >}}
{{% expand "<span class='req'>Recommended</span> – Set appropriate directory permissions" %}}

<!-- Directory permissions content -->

{{% /expand %}}
{{% expand "<span class='req'>Recommended</span> – Verify the authenticity of downloaded binary" %}}

<!-- Binary authenticity content -->

{{% /expand %}}
{{< /expand-wrapper >}}
```

    If you do not move the `influxd` binary into your `$PATH`, prefix the executable
    `./` to run it in place.

{{< expand-wrapper >}}
{{% expand "<span class='req'>Recommended</span> – Set appropriate directory permissions" %}}

> For added security, it is recommended to set the permissions on the influxdb data-dir to not be world readable to prevent unwanted access to data.

Reviewer: ```suggestion
To prevent unwanted access to data, we recommend setting the permissions on the influxdb `data-dir` to not be world readable.
```

Example:

```shell
> chmod 0750 ~/.influxdbv2
```

> {{% /expand %}}

Reviewer 2: Move this into a single `expand-wrapper` block as explained in the other comment.

> mv 'C:\Program Files\InfluxData\influxdb2-{{< latest-patch >}}-windows-amd64' 'C:\Program Files\InfluxData\influxdb'
```

{{< expand-wrapper >}}
{{% expand "<span class='req'>Recommended</span> – Set appropriate directory permissions" %}}

> For added security, it is recommended to set the permissions on the influxdb data-dir to not be world readable to prevent unwanted access to data.

Reviewer: ```suggestion
To prevent unwanted access to data, we recommend setting the permissions on the influxdb `data-dir` to not be world readable.
```
